packages/unplugin-typia/src/core/typia.ts

```diff
@@ -1,10 +1,11 @@
+import type { Alias } from 'vite';
 import ts from 'typescript';
-import { resolve } from 'pathe';
+import path, { resolve } from 'pathe';
```

Comment:
I don't want to use named export for 'pathe'
|'d like to fix it!

Reply:
Yes, this was due to IDE's auto imports as well 👍🏻